src/toastfile.rs

```diff
@@ -12,6 +13,67 @@ pub const DEFAULT_LOCATION: &str = "/scratch";
 // The default user for commands and files copied into the container
 pub const DEFAULT_USER: &str = "root";
 
+#[derive(Clone, Debug, Eq, PartialEq)]
```

Comment:
This is silly, but would you mind adding a comment above this for consistency with the other struct definitions:

```
// This struct represents a path on the host and a corresponding path in the container.
#[derive(Clone, Debug, Eq, PartialEq)]
pub struct MappingPath {
```